spk/links/Makefile

```diff
@@ -0,0 +1,21 @@
+SPK_NAME = links
+SPK_VERS = 2.12
+SPK_REV = 1
+SPK_ICON = src/links.png
+
+DEPENDS = cross/links
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Links is a web browser running in text mode.
+DISPLAY_NAME = Links
+BETA = 1
+
+HOMEPAGE   = http://links.twibright.com/
```

Comment:
Gotcha, based these files off of a different package, I should have fixed the alignment but it slipped by my eyes.